cclib/parser/psi3parser.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Parser for Psi3 and Psi4 output files"""
+
+
+import re
+
+import numpy
+
+from cclib.parser import data
+from cclib.parser import logfileparser
+from cclib.parser import utils
+
+
+class Psi3(logfileparser.Logfile):
+    """A Psi3 log file."""
+
+    def __init__(self, *args, **kwargs):
+
+        # Call the __init__ method of the superclass
+        super(Psi3, self).__init__(logname="Psi3", *args, **kwargs)
+
+    def __str__(self):
+        """Return a string representation of the object."""
+        return "Psi3 log file %s" % (self.filename)
+
+    def __repr__(self):
+        """Return a representation of the object."""
+        return 'Psi3("%s")' % (self.filename)
+
+    def normalisesym(self, label):
+        """Psi3 does not require normalizing symmetry labels."""
+        return label
+
+    def extract(self, inputfile, line):
+        """Extract information from the file object inputfile."""
+
+        self.metadata["package_version"] = 3
```

Comment:
Do we need this bit of metadata, if we know this is Psi3?

Reply:
I would like most `metadata` keys to be present for each parser, especially ones to be compliant with the provenance part of QCJSON. A different PR should set `package_version` to the full version (3.4.0 for ours), not just the major version.

Reply:
Good point. as long `3` correspond to the actual version of Psi3.